iosxr_iso2vbox.py

```diff
@@ -124,10 +149,10 @@ def main(argv):
     # Handle Input ISO (Local or URI)
     if re.search(':/', args.ISO_FILE):
         # URI Image
-        cmdstring = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
+        cmd_string = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
```

Comment:
I don't care that much - it was just fyi

On 7/14/16, 3:45 PM, Richard Wellum wrote:
>> @@ -124,10 +149,10 @@ def main(argv):
>>       # Handle Input ISO (Local or URI)
>>       if re.search(':/', args.ISO_FILE):
>>           # URI Image
>> -        cmdstring = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
>> +        cmd_string = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
>
> .format is less readable to me. But maybe .format handles off characters in the scp string better.
>
> I'll look into this - probably open a us.
>
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/ios-xr/iosxrv-x64-vbox/pull/10/files/fc29a814958e64ddff6a567916527f1c4084fe76#r70870505
>